core/renderers/geras/renderer.js

```diff
@@ -65,17 +60,56 @@ Blockly.utils.object.inherits(Blockly.geras.Renderer,
  */
 Blockly.geras.Renderer.prototype.init = function() {
   Blockly.geras.Renderer.superClass_.init.call(this);
-  this.highlightConstants_ = new Blockly.geras.HighlightConstantProvider();
+  this.highlightConstants_ = this.makeHighlightConstants_();
 };
 
 /**
- * Get the renderer's highlight constant provider.
- * @return {Blockly.geras.HighlightConstantProvider} The highlight constant
+ * Create a new instance of the renderer's render info object.
+ * @param {!Blockly.BlockSvg} block The block to measure.
+ * @return {!Blockly.geras.RenderInfo} The render info object.
+ * @protected
```

Comment:
Do you need @override on the make<foo> methods for the specific renderers?

Reply:
Yes!